### PR TITLE
Reduce openzipkin links to just home page and allow a user-configurab…

### PR DESCRIPTION
**Pull request by anuraaga**
…le support link.

Discussion in #2375

After (with supportUrl defined)
image

After (without supportUrl defined)
image

Before
image

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
maybe before and after image to make it more clear?

File: zipkin-lens/src/translations/es/messages.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "End Time": "Tiempo de Fin",
   "Failed to load this file": "Fallo al cargar este archivo",
   "Filter": "Filtro",
+  "Help": "",
```
Comment (jeqo) [inline zipkin-lens/src/translations/es/messages.json]:
```suggestion
  "Help": "Ayuda",
```

**Review by shakuzen — APPROVED** [review]
Looks good to me. Thanks for making the changes! Could we try to get a review from the user who gave the initial feedback?

**Comment by codefromthecrypt** [comment]
Should we use English "Help" or "Support" ? I tend to think of the word support, and whatever we put in here lives in config, so figured worth a short bikeshed.

As far as I can tell in the code, by default we don't add a default now. Maybe a screenshot of default (help/support defined) can ... help :)

@jcarres-mdsol does this make sense for configuring your slack url?
@Monkeyanator thoughts on the reduced UI clutter (for lack of better word)?

**Comment by anuraaga** [comment]
Was hoping to leave the other screenshot to imagination ;) But added it.

Have no opinion at all on `help` or `support` - if no one else votes on it I'll rename to support.

**Comment by codefromthecrypt** [comment]
I like the without visually as it balances the white icons top and bottom. Nice!

File: zipkin-lens/src/translations/es/messages.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
   "End Time": "Tiempo de Fin",
   "Failed to load this file": "Fallo al cargar este archivo",
   "Filter": "Filtro",
+  "Help": "Ayuda",
```
Comment (jeqo) [inline zipkin-lens/src/translations/es/messages.json]: if changing to "Support": 

```suggestion
  "Help": "Soporte",
```

**Review by jeqo — APPROVED** [review]

**Comment by jcchavezs** [comment]
+1

**Comment by anuraaga** [comment]
Renamed from help -> support

**Comment by codefromthecrypt** [comment]
Good job team. thanks for driving this home @anuraaga!

**Pull request merged**